catalog/core/catalog-core-impl/filter-proxy/src/test/java/ddf/catalog/filter/proxy/adapter/test/FilterAdapterTest.java

```diff
@@ -95,25 +96,25 @@
             new DefaultPeriodDuration(DAY_IN_MILLISECONDS);
 
     @Test
-    public void includeFilter() {
+    public void testIncludeFilter() {
```

Comment:
Our Unit Test guidelines state that the test prefix is optional.  The majority of the unit tests within this package are prefixed with test, so this was changed for continuity and uniformity.
https://codice.atlassian.net/wiki/display/DDF/Unit+Testing